backend/src/main/java/com/github/lenaruppert/backend/service/JobService.java

```diff
@@ -56,4 +57,18 @@ public List<Job> getJobsByClientId(String clientId) {
     public Job getJobById(String id) {
         return jobRepository.findById(id).orElseThrow(NoSuchElementException::new);
     }
+
+    public Job deleteJobById(String id) {
+        Optional<Job> jobToDelete = jobRepository.findById(id);
+        if (!jobToDelete.isPresent()) {
+            throw new NoSuchElementException(id);
+        }
+        Job job = jobToDelete.get();
```

Comment:
```suggestion
        Job job = getJobById(id);
```
This includes already the exception if the job doesn't exist and is a bit shorter :)